argus/scenario.py

```diff
@@ -418,6 +418,61 @@ def get_remote_client(self, username=None, password=None,
     remote_client = util.cached_property(get_remote_client, 'remote_client')
 
 
+class RescueWindowsScenario(BaseWindowsScenario):
+    """Instance rescue Windows-based scenario."""
+
+    def rescue_server(self):
+        admin_pass = self._image.default_ci_password
+        self._servers_client.rescue_server(self._server['id'],
+                                           adminPass=admin_pass)
+        self._servers_client.wait_for_server_status(self._server['id'],
+                                                    'RESCUE')
+
+    def unrescue_server(self):
+        self._servers_client.unrescue_server(self._server['id'])
+        self._servers_client.wait_for_server_status(self._server['id'],
+                                                    'ACTIVE')
+
+    def run_test_suite(self):
+        testloader = unittest.TestLoader()
+        suite = unittest.TestSuite()
+        for test_class in self._test_classes:
+            testnames = testloader.getTestCaseNames(test_class)
+            for name in testnames:
+                suite.addTest(test_class(name,
+                                         manager=self,
+                                         service_type=self._service_type,
+                                         image=self._image))
+        self._result = unittest.TestResult()
```

Comment:
Why don't you use the Test result set in the constructor? This also hardcodes the test result to unittest.TestResult.